website/scripts/build-static-content.js

```diff
@@ -512,6 +512,160 @@ module.exports = {
+                    fleetColumn.hidden = false;
+                  }
+                  if(columnHasFleetOverrides.required !== undefined) {
+                    fleetColumn.required = _.clone(columnHasFleetOverrides.required);
+                  }
+                  mergedTableColumns.push(fleetColumn);
+                }
+              }
+            }
+            expandedTableToPush.columns = mergedTableColumns;
+          }
+          expandedTables.push(expandedTableToPush);
+        }
+
+        // After we've gone through the tables in the Osquery schema, we'll go through the tables in the Fleet schema JSON, and add any tables that don't exist in the osquery schema.
+        for (let fleetOverridesForTable of fleetOverridesForTables) {
+          let fleetSchemaTableExistsInOsquerySchema = _.find(rawOsqueryTables, (table)=>{
+            return fleetOverridesForTable.name === table.name;
+          });
+          if(!fleetSchemaTableExistsInOsquerySchema) { // If a table in the Fleet schema does not exist in the osquery schema, we'll add it to the final schema.
+            expandedTables.push(fleetOverridesForTable);
+          }
+        }
+
+        // Once we have our merged schema, we'll create ejs partials for each table.
+        for(let table of expandedTables) {
+          let keywordsForSyntaxHighlighting = [];
+          keywordsForSyntaxHighlighting.push(table.name);
+          if(!table.hidden) { // If a table has `"hidden": true` the table won't be shown in the final schema, and we'll ignore it
+            // Start building the markdown string for this table.
+            let tableMdString = '\n## '+table.name;
+            if(_.endsWith(table.name, '_events') || table.evented){
+              // If the table name ends in `_events` or has `"evented": true`, we'll add an evented table label (in html)
+              tableMdString += '   <div purpose="evented-table-label"><span>EVENTED TABLE</span></div>\n';
+            }
+            // Add the tables description to the markdown string and start building the table in the markdown string
+            tableMdString += '\n\n'+table.description+'\n\n|Column | Type | Description |\n|-|-|-|\n';
+
+            // Iterate through the columns of the table, we'll add a row to the markdown table element for each column in this schema table
+            for(let column of table.columns) {
+              if(!column.hidden) { // If te column is hidden, we won't add it to the final table.
+                let columnDescriptionForTable = column.description;// Set the initial value of the description that will be added to the table for this column.
+                if(typeof columnDescriptionForTable === 'String'){ // Replacing pipe characters with an html entity in column descriptions to keep it from breaking markdown tables.
```

Comment:
typeof returns `'string'` with a lowercase "s"

Reply:
Better yet, instead of doing this check here, move this code up make sure it's always a string (default to empty string) in the Fleet schema